viz_scripts/scaffolding.py

```diff
@@ -311,29 +349,111 @@ def energy_impact_kWH(df,distance):
     df['Energy_Impact(kWH)']  = round((df['Replaced_mode_EI(kWH)'] - df['Mode_confirm_EI(kWH)']),3)
     return df
 
-def CO2_footprint_lb(df, distance, col):
+def CO2_footprint_default(df, distance, col):
     """ Inputs:
     df = dataframe with data
     distance = distance in miles
     col = Replaced_mode or Mode_confirm
     """
+
+    conversion_lb_to_kilogram = 0.453592 # 1 lb = 0.453592 kg
+    conversion_mile_to_kilometer = 1.60934 # 1 mile = 1.60934 km
```

Comment:
given https://github.com/e-mission/em-public-dashboard/pull/91/files#r1380656095 we don't need this, right?

Reply:
I have removed the mile to kilometer conversion constant, since it's not being used. I have kept the `conversion_lb_to_kilogram` because the unit of CO2 emission is `lb/Trip` or `kg/Trip`. So, have kept the conversion between `lb` to `kilogram` will come handy in future, for instance where we have dynamic config but want to represent the emission in `Imperial system` i.e. `lb/ Trip` emission.